client/src/components/Header/Header.js

```diff
@@ -22,7 +28,12 @@ const Sidebar = ({ onClose }) => {
           action={() => navigate("/task_history")}
         />
         <SidebarItem label={"Insights"} action={() => navigate("/insights")} />
-        <SidebarItem label={"Log out"} action={handleLogOut} />
+        <SidebarItem
+          label={"Log out"}
+          action={async () => {
```

Comment:
No need for async here since `handleLogout` is already an async function

Reply:
Removed `async` keyword.